docs/_writing-tests/general-guidelines.md

```diff
@@ -98,6 +98,30 @@ This is not, however, to discourage testing of edge cases or
 interactions between multiple features; such tests are an essential
 part of ensuring interoperability of the web platform.
 
+Tests should pass when the feature under test exposes the expected behavior,
+and they should fail when the feature under test is not implemented or is
+implemented incorrectly. Tests should not rely on unrelated features if doing
+so causes failures in the latest stable release of [Apple
+Safari][apple-safari], [Google Chrome][google-chrome], [Microsoft
```

Comment:
Should we talk about rendering engines, here, instead of browsers?

Reply:
Rendering engines don't have releases, so that wouldn't quite work.